extensions/azurePublishNew/src/node/availableResources.ts

```diff
@@ -27,6 +27,15 @@ export type AppRegistrationResourceConfig = ResourceConfig & {
   appName: string;
 };
 
+type ApplicationType = 'web' | 'other';
+
+export type AppInsightsResourceConfig = ResourceConfig & {
```

Comment:
As per previous reviews, the config types should be in the modules that own them.  That way availableResource.ts has dependencies on individual resource files, but the individual files don't also depend on availableResource.ts (i.e. a circular dependency).

Reply:
fixed